tests/ConfigurationTest.cpp

```diff
@@ -11,7 +11,7 @@ BOOST_AUTO_TEST_CASE(ReadConfiguration)
   const std::string                     expectedMatrixFileName{"../data/m3.csv"};
   const int                             expectedMatrixSize{3};
 
-  Configuration configuration{"testconfig.yml"};
+  Configuration configuration{"../tests/testconfig.yml"};
```

Comment:
This should not be necessary as you use `WORKING_DIRECTORY` in `CMakeLists.txt`.